roles/collect/defaults/main.yml

```diff
@@ -1,2 +1,17 @@
 ---
 # defaults file for collect
+collect_endpoint_scheme: 'http'
+collect_endpoint_host: 'localhost'
+collect_endpoint_port: '8001'
+collect_endpoint_path: 'api/v1/namespaces/metering/services/https:reporting-operator:http/proxy/api/v1/reports/get'
+collect_format: 'csv'
+collect_report_name: 'pod-cpu-usage'
+collect_endpoint_params: '?name={{ collect_report_name }}&format={{ collect_format }}'
+collect_archive_format: bz2
+collect_archive_path: '/tmp'
```

Comment:
Could use the following instead of just `/tmp`:
https://docs.ansible.com/ansible/devel/reference_appendices/config.html#default-local-tmp

Reply:
This is the path where we write the tarball out. My expectation is that it should be overridden by the caller.

Reply:
that's fair, just trying to limit what someone would need to provide and utilize any pre-configured ansible setup.